actionpack/test/controller/request_forgery_protection_test.rb

```diff
@@ -987,6 +1005,10 @@ def assert_presence_and_fetch_form_csrf_token
       end
     end
 
+    def assert_no_cache
+      assert_equal "no-cache", @response.headers["Cache-Control"]
```

Comment:
Can you inline those assertions? We prefer code readability over DRY on tests.

Reply:
Done